dlt/normalize/items_normalizers.py

```diff
@@ -52,6 +63,120 @@ def __init__(
     def __call__(self, extracted_items_file: str, root_table_name: str) -> List[TSchemaUpdate]: ...
 
 
+class ModelItemsNormalizer(ItemsNormalizer):
+    def _add_or_replace_column(
+        self,
+        parsed_select: sqlglot.exp.Select,
+        column_name: Literal["_dlt_load_id", "_dlt_id"],
+        alias_expression: sqlglot.exp.Alias,
+        root_table_name: str,
+    ) -> Optional[TSchemaUpdate]:
+        """
+        Adds or replaces a column in the SELECT clause of a parsed SQL query and
+        returns a single TSchemaUpdate if a new column was added, otherwise None.
+        """
+        schema = self.schema
+        schema_update: TSchemaUpdate = {}
+
+        column_replaced = False
+        for i, select in enumerate(parsed_select.selects):
+            # Check if the column already exists
+            select_alias = (
+                select.alias_or_name.lower()
+                if isinstance(select, sqlglot.exp.Alias)
+                else (
+                    select.output_name.lower() if isinstance(select, sqlglot.exp.Column) else None
+                )
+            )
+
+            if select_alias == column_name:
+                # Replace the existing column
+                parsed_select.selects[i] = alias_expression
+                column_replaced = True
```

Comment:
small note: you can just return None here, then you do not need column_replace and the if clause further down